test/fluree/server/integration/basic_transaction_test.clj

```diff
@@ -64,6 +64,51 @@
       (is (= {"ledger" ledger-name, "t" 2}
              (-> res :body json/read-value (select-keys ["ledger" "t"])))))))
 
+(deftest ^:integration ^:json transaction-with-dbs-at-multiple-t-values
```

Comment:
I think this test could use more English commentary through more nested `testing` macros and/or using the third, `msg` argument of the `is` macro.